README.md

```diff
@@ -45,17 +45,33 @@ with more configuration options.
 From within the root of the repo run the following command to build an image of
 `renterd` tagged `renterd`.
 
-```bash
-docker build -t renterd .
+### Production
+
+```sh
+docker build --build-arg BUILD_TAGS='netgo' -t renterd:latest -f ./docker/Dockerfile .
```

Comment:
You don't need the `BUILD_TAGS` for the production build. It's probably easier to leave them out.